Sources/PrimerSDK/Classes/Core/Checkout Components/PrimerHeadlessUniversalCheckout.swift

```diff
@@ -25,13 +25,8 @@ public class PrimerHeadlessUniversalCheckout {
             PrimerHeadlessUniversalCheckout.current.delegate = delegate
         }
         
-        guard PrimerHeadlessUniversalCheckout.current.delegate != nil else {
-            let err = PrimerError.missingPrimerCheckoutComponentsDelegate(userInfo: ["file": #file, "class": "\(Self.self)", "function": #function, "line": "\(#line)"])
-            ErrorHandler.handle(error: err)
-            DispatchQueue.main.async {
-                completion(nil, err)
-            }
-            return
+        if PrimerHeadlessUniversalCheckout.current.delegate == nil {
+            print("WARNING: PrimerHeadlessUniversalCheckout delegate has not been set, and you won't be able to receive the Payment Method Token data to create a payment.")
```

Comment:
Is there a specific reason for raising the Warning and not the ErrorHandler? I believe e I asked before but just wanted a 🧠 refresh. Especially as a similar  `print` is added in another change alongside the ErrorHandler

Reply:
We don't want to break the flow if someone doesn't use the HUC, but somehow calls something from the HUC